src/gcodeExport.cpp

```diff
@@ -197,6 +201,7 @@ std::string GCodeExport::getFileHeader(const double* print_time, const std::vect
         else if (flavor == EGCodeFlavor::REPRAP)
         {
             prefix << ";Filament used: " << ((filament_used.size() >= 1)? filament_used[0] / (1000 * extruder_attr[0].filament_area) : 0) << "m" << new_line;
+            prefix << ";Layer height: " << layer_height << new_line;
```

Comment:
The `layer_height` variable should be declared and defined just above this line.

Note that when you are printing in one-at-a-time mode you can have different layer heights per meshgroup, so this comment is kind of deceiving.

Perhaps it is indeed better to add this comment in the starting gcode of your printer definition.